messages/dev.generate.flag.md

```diff
@@ -158,60 +158,60 @@ Do you want to use the standard definition (character, summary, description, def
 
 # flagDescriptions.duration
 
-a unit and quantity of time (example: X minutes, 4 days). You can specify min/max/default.
+A unit and quantity of time, such as "10 minutes" or "4 days". You can optionally specify the minimum, maximum, or default values to take advantage of the built-in CLI validation rules.
 
 # flagDescriptions.option
 
-a choice from a defined list of string options. You can set the allowed values.
+Value is a single choice out of a pre-defined list of string values. You will then be asked for the list of allowed values.
 
-# flagDescriptions.integer
+# flagDescriptions.integer 
 
-an integer with built-in validation of optional min/max/default values.
+An integer. You can optionally specify the minimum, maximum, or default values to take advantage of the built-in CLI validation rules.
 
 # flagDescriptions.custom
 
-a flag with a custom typescript type. You'll need to add some code after the scaffolding is complete.
+Flag with a custom typescript type. You must add some code after the scaffolding is complete.
 
-# flagDescriptions.salesforceId
+# flagDescriptions.salesforceId 
 
-a valid salesforce record ID. You can specify the 3-character prefix and/or the length of the ID.
+A valid Salesforce record ID. You can specify the 3-character prefix and the length of the ID.
 
 # flagDescriptions.file
 
-a local file path. You can specify whether the file must exist.
+A local file. You can specify whether the file must already exist, or if the command will create it if necessary.
```

Comment:
there's nothing in the flag for creating files.  A command can create a file, but that's unrelated to the flag

Reply:
same comment for `directory` below

Reply:
Oh! gotcha, thanks.